imports/ui/components/SelectBar.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Dropdown } from "semantic-ui-react";
+import "semantic-ui/dist/semantic.min.css";
+
+const groceryOptions = [
+  {
+    key: "Bananas",
+    text: "Bananas",
+    value: "Bananas"
+  },
+  {
+    key: "Apples",
+    text: "Apples",
+    value: "Apples"
+  },
+  {
+    key: "Oranges",
+    text: "Oranges",
+    value: "Oranges"
+  }
+];
+
+const DropdownExampleSelection = () => (
+  <Dropdown
+    placeholder="Select..."
+    fluid
+    search
+    selection
+    options={groceryOptions}
+  />
+);
+
+export default DropdownExampleSelection;
```

Comment:
Dropdown of items using semantic-ui instead.

Pros: Much easier to read & interpret and use it in the future.
Cons: As you can see, it installs a lot of dependencies for semantic-ui.

@nicolehli @YaoLiu93 Let me know what you guys prefer